server/src/main/java/org/apache/druid/segment/handoff/CoordinatorBasedSegmentHandoffNotifier.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.druid.query.SegmentDescriptor;
 import org.apache.druid.server.coordination.DruidServerMetadata;
 
+import org.joda.time.Duration;
```

Comment:
```suggestion
import org.joda.time.Duration;

```

Reply:
this is causing checkstyle to fail.

Reply:
Thanks for the review @kfaraz! Just pushed this fix